Makefile.am

```diff
@@ -27,13 +27,13 @@ pkgconfig_DATA 	= squashfuse.pc
 # Main library: libsquashfuse
 libsquashfuse_la_SOURCES = swap.c cache.c table.c dir.c file.c fs.c \
 	decompress.c xattr.c hash.c stack.c traverse.c util.c \
-	nonstd-pread.c nonstd-stat.c \
+	nonstd-pread.c nonstd-stat.c ops.c \
 	squashfs_fs.h common.h nonstd-internal.h nonstd.h swap.h cache.h table.h \
 	dir.h file.h decompress.h xattr.h squashfuse.h hash.h stack.h traverse.h \
-	util.h fs.h
+	util.h fs.h ops.h
 libsquashfuse_la_CPPFLAGS = $(ZLIB_CPPFLAGS) $(XZ_CPPFLAGS) $(LZO_CPPFLAGS) \
 	$(LZ4_CPPFLAGS) $(ZSTD_CPPFLAGS)
-libsquashfuse_la_LIBADD =
+libsquashfuse_la_LIBADD = libfuseprivate.la $(FUSE_LIBS)
```

Comment:
We could try to build and link without it! This may have been something we added at a time when we weren't linking correctly and thought we needed it ':D

Reply:
So I'm noticing that I'm able to build and link to the generated static libraries, and run our code that depends on the libraries. but there are issues when trying to build the .so libraries in squashfuse, saying that there's an undefined reference to functions located in libfuseprivate. I imagine this is because the ops code was originally in the source file which linked both libfuseprivate and libsquashfuse. 
TLDR; I think for our purposes we don't need that line of code, but for squashfuse to function all around I think we do.